src/dev/build/tasks/os_packages/docker_generator/templates/ironbank/hardening_manifest.yaml

```
@@ -11,7 +11,7 @@ tags:
 # Build args passed to Dockerfile ARGs
 args:
   BASE_IMAGE: 'redhat/ubi/ubi9'
-  BASE_TAG: "9.6"
+  BASE_TAG: "\"9.7\""
```

@v1v it seems like the string replacement is incorrect now?

---

This will be solved as soon as https://github.com/elastic/kibana/pull/242980 gets merged.
Future PRs should not have that particular misbehaviour - it happened when using a new version of `updatecli`.

For now let's fix it here


```suggestion
  BASE_TAG: "9.7"
```